extensions/ql-vscode/src/extension.ts

```diff
@@ -1010,6 +1013,33 @@ async function activateWithInstalledDistribution(
   };
 }
 
+/**
+ * Handle changes to the external config file. This is used to restart the query server
+ * when the user changes options.
+ * See https://docs.github.com/en/code-security/codeql-cli/using-the-codeql-cli/specifying-command-options-in-a-codeql-configuration-file#using-a-codeql-configuration-file
+ */
+function watchExternalConfigFile(app: ExtensionApp, ctx: ExtensionContext) {
+  if (process.env.HOME) {
```

Comment:
Do we not want to have this work on Windows? It seems like we can call [`os.homedir()`](https://nodejs.org/docs/latest-v16.x/api/os.html#oshomedir) to get the home directory in a platform independent way.

Reply:
Ah...right. I forgot about that. I will fix.